src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -1,67 +0,0 @@
-﻿using System;
-using System.Reflection;
-using System.Resources;
-using System.Runtime.InteropServices;
-
-// General Information about an assembly is controlled through the following 
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyCompany("Autodesk, Inc")]
-[assembly: AssemblyProduct("Dynamo")]
-[assembly: AssemblyCopyright("Copyright © Autodesk, Inc 2017")]
-[assembly: AssemblyTrademark("")]
-
-//In order to begin building localizable applications, set 
-//<UICulture>CultureYouAreCodingWith</UICulture> in your .csproj file
-//inside a <PropertyGroup>.  For example, if you are using US english
-//in your source files, set the <UICulture> to en-US.  Then uncomment
-//the NeutralResourceLanguage attribute below.  Update the "en-US" in
-//the line below to match the UICulture setting in the project file.
-
-[assembly: NeutralResourcesLanguage("en-US", UltimateResourceFallbackLocation.Satellite)]
-
-// Make it easy to distinguish Debug and Release (i.e. Retail) builds;
-// for example, through the file properties window.
-#if DEBUG
-[assembly: AssemblyConfiguration("Debug")]
-[assembly: AssemblyDescription("Flavor=Debug")] // a.k.a. "Comments"
-#else
-[assembly: AssemblyConfiguration("Release")]
-[assembly: AssemblyDescription("Flavor=Release")] // a.k.a. "Comments"
-#endif
-
-[assembly: CLSCompliant(true)]
-
-// Setting ComVisible to false makes the types in this assembly not visible 
-// to COM components.  If you need to access a type in this assembly from 
-// COM, set the ComVisible attribute to true on that type.
-[assembly: ComVisible(false)]
-
-// Note that the assembly version does not get incremented for every build
-// to avoid problems with assembly binding (or requiring a policy or
-// <bindingRedirect> in the config file).
-//
-// The AssemblyFileVersionAttribute is incremented with every build in order
-// to distinguish one build from another. AssemblyFileVersion is specified
-// in AssemblyVersionInfo.cs so that it can be easily incremented by the
-// automated build process.
-[assembly: AssemblyVersion("2.2.0.4207")]
-
-
-// By default, the "Product version" shown in the file properties window is
-// the same as the value specified for AssemblyFileVersionAttribute.
-// Set AssemblyInformationalVersionAttribute to be the same as
-// AssemblyVersionAttribute so that the "Product version" in the file
-// properties window matches the version displayed in the GAC shell extension.
-//[assembly: AssemblyInformationalVersion("1.0.0.0")] // a.k.a. "Product version"
-// Version information for an assembly consists of the following four values:
-//
-//      Major Version
-//      Minor Version 
-//      Build Number
-//      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers 
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.2.0.4207")]
```

Comment:
you need to revert this change - this file should not be removed.

Reply:
I already reverted it, and I added it like it was at the beginning of this PR.